analyses/chromosomal-instability/01-localization-of-breakpoints.nb.html

```diff
@@ -3444,7 +3444,7 @@ <h3>Session Info</h3>
 loaded via a namespace (and not attached):
  [1] Rcpp_1.0.1             pillar_1.4.2           compiler_3.6.0        
  [4] GenomeInfoDb_1.22.1    XVector_0.26.0         base64enc_0.1-3       
- [7] bitops_1.0-6           tools_3.6.0            zlibbioc_1.32.0       
+ [7] bitops_1.0-6           tools_3.6.0            zlibbioc_1.30.0       
```

Comment:
This diff suggests a versioning/Docker issue. The version I see in your new code here committed here is _correct_ for the Docker, so the data difference may be because someone previously ran outside of Docker.

Reply:
Thanks for looking into this! Just to confirm, I did run within Docker for this commit (although like Jaclyn mentioned, I initially had issues getting Docker set up, but I later figured it out).